css/properties/-webkit-text-security.json

```
@@ -33,7 +33,7 @@
           },
           "status": {
             "experimental": false,
-            "standard_track": false,
+            "standard_track": true,
```

@dipikabh, I think that you're thinking about the "experimental" status.  Standard track means that the property is defined in a spec, but that's not the case.
Apologies @andreydanil, would you mind reverting that change?

```suggestion
            "standard_track": false,
```